projects/storefrontlib/src/cms-components/product/product-details/product-summary/product-summary.component.ts

```diff
@@ -76,22 +77,21 @@ export class ProductSummaryComponent implements OnInit {
       .subscribe(reviewsTabLabel => {
         const tabsComponent = this.getTabsComponent();
         const reviewsTab = this.getTabByLabel(reviewsTabLabel, tabsComponent);
-
         const reviewsComponent = this.getReviewsComponent();
-
         if (reviewsTab && reviewsComponent) {
           this.clickTabIfInactive(reviewsTab);
-          reviewsComponent.scrollIntoView();
+          setTimeout(
```

Comment:
After trigger click tab, there is a very small delay before it appear. Synchronous code was calling scrollIntoView before object was available to scrolling into it.

Another idea is to use e.g waiting for reviewsComponent.getClientRects().length > 0

Reply:
One option could be to use the stores to await responses and it might be more correct not to be using "click" at all in ts code. But maybe that's something that can be done if the issue comes up again since its quite a refactor for very little gain and it already works now.